src/main/java/bot/command/Command.java

```diff
@@ -0,0 +1,19 @@
+package bot.command;
+
+import bot.exception.FileErrorBotException;
+import bot.exception.IllegalExpressionBotException;
+
+import java.io.IOException;
+
+public abstract class Command {
+
+    public static final String SPACER = "____________________________________________________________";
```

Comment:
Since spacer is only used in commands, you might consider making it protected to limit the use of spacer to within subclasses.